src/test/java/seedu/address/storage/JsonAdaptedEventTest.java

```diff
@@ -0,0 +1,172 @@
+package seedu.address.storage;
+
+import static java.time.temporal.TemporalAdjusters.next;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.storage.JsonAdaptedEvent.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalSchedule.SE_TUTORIAL;
+
+import java.time.DayOfWeek;
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.recurfrequency.RecurFrequency;
+import seedu.address.model.schedule.Event;
+import seedu.address.model.schedule.EventDescription;
+
+public class JsonAdaptedEventTest {
+    private static final String INVALID_EVENT_DESCRIPTION = "";
+    private static final String INVALID_DATE = "20-12-2022";
+    private static final String INVALID_TIME = "09:000";
+    private static final String INVALID_DURATION = "8HdM";
+    private static final String INVALID_RECURFREQUENCY = "non-recurring";
+
+    private static final String VALID_EVENT_DESCRIPTION = SE_TUTORIAL.getEventDescription().toString();
+    private static final String VALID_DATE = SE_TUTORIAL.getDate().toString();
+    private static final String VALID_TIME = SE_TUTORIAL.getTime().toString();
+    private static final String VALID_DURATION = SE_TUTORIAL.getDuration().toString();
+    private static final String VALID_RECURFREQUENCY = SE_TUTORIAL.getRecurFrequency().toString();
```

Comment:
This looks a bit weird. We can just use the data in EventBuilder. And use the `new EventBuilder().build()` to create the event.

Reply:
Hmm, we are trying to test `JsonAdaptedEvent` here so we want to create the `Event` through `JsonAdaptedEvent::toModelType`. Could you clarify what the `EventBuilder` should be used for?

Reply:
`EventBuilder` allows us to edit an attribute of `Event` without having to put all the inputs into the `Event` constructor. This is done by all the `with...` methods. And the when you are done editing the `Event`, you can just use the `EventBuilder#build` method to create the Event.